apps/site/lib/site_web/templates/alert/_item.html.eex

```diff
@@ -18,6 +18,9 @@
       </div>
       <div>
         <%= replace_urls_with_links(@alert.header) %>
+        <%= if @alert.url != nil do %>
+          <%= replace_urls_with_links(@alert.url) %>
```

Comment:
@alicia-mbta will `replace_urls_with_links` do the right thing in case of an `@alert.url` of `""`? if not, let's check for that in the conditional on line 21 as well.